microsoft-365/commerce/subscriptions/what-if-my-subscription-expires.md

```diff
@@ -198,6 +198,10 @@ At this point, you can't reactivate the subscription. However, as a global or bi
 > [!NOTE]
 > Adding a new subscription of the same type that has been deprovisioned does not restore the data that was associated with the deprovisioned subscription.
 
+
+> [!NOTE]
+> If CSP license is supended, there is no 30 day grace period, services are disabled immediately. Data will be deleted after 90 days if tenant is not reactivated by adding new licenses or reactivating CSP licenses.
```

Comment:
```suggestion
> If a CSP license is suspended, there is no 30 day grace period, and services are disabled immediately. Data will be deleted after 90 days if the tenant is not reactivated by adding a new license or by reactivating the CSP license.
```

Reply:
About _["...or by reactivating the CSP license."]_ This is confusing. You do not necessarily have to acquire a CSP license, it can be with another contract model (Web, EA, MPSA, ...)

Reply:
That makes sense. Removed the CSP licenses part.